openmetadata-docs/content/v0.13.3/connectors/ingestion/workflows/dbt/ingest-dbt-descriptions.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Ingest Descriptions from dbt
+slug: /connectors/ingestion/workflows/dbt/ingest-dbt-descriptions
+---
+
+# Ingest Descriptions from dbt
+
+Ingest the descriptions from dbt `manifest.json` or `catalog.json` file into OpenMetadata tables and columns.
+
+{% note %}
+By default descriptions from `manifest.json` will be imported. Descriptions from `catalog.json` will only be updated if `catalog.json` file is passed.
```

Comment:
maybe silly question: can we have descriptions in both manifest and catalog? If so, which one are we persisting?

Reply:
We can have descriptions in both files
If only manifest file is passed descriptions from manifest will be updated
If manifest and catalog both are passed descriptions from catalog will be updated

Reply:
then let's add this info here

Reply:
Added the info